pip/wheel.py

```diff
@@ -692,6 +697,18 @@ def __build_one(self, req, tempd):
             logger.error('Failed building wheel for %s', req.name)
             return False
 
+    def _clean_one(self, req):
+        base_args = self._base_setup_args(req)
+
+        logger.info('Running setup.py clean for %s', req.name)
+        clean_args = base_args + ['clean', '--all']
+        try:
+            call_subprocess(clean_args, cwd=req.source_dir, show_stdout=False)
+            return True
+        except:
+            logger.error('Failed cleaning build dir for %s', req.name)
```

Comment:
Yeah, I was also debating the logging level with myself.  I ended up going with "a setup.py command has failed, so call it an error", which seems to be the rule in other places.  I'm happy to go with whatever consensus, though.